.github/workflows/deploy.yml

```diff
@@ -98,6 +98,12 @@ jobs:
       run: |
         echo "TAG=${{ github.sha }}" >> $GITHUB_ENV
 
+    - name: Bump version and push tag
```

Comment:
If this is inside the matrix, it will generate a new tag for each environment. Can it be moved to the build job and the tag provided as output?